content/en/docs/distributions/gke/customizing-gke.md

```diff
@@ -156,78 +157,131 @@ options:
-to `false`:
+Create the [ContainerNodePool](https://cloud.google.com/config-connector/docs/reference/resource-docs/container/containernodepool) resource adopting GPU, for exmaple, create a new file `containernodepool-gpu.yaml` file and fulfill the value below:
 
 ```
-    ...
-    clusterAutoscaling:
-      enabled: false
-      autoProvisioningDefaults:
-    ...
+apiVersion: container.cnrm.cloud.google.com/v1beta1
+kind: ContainerNodePool
+metadata:
+  labels:
+    kf-name: KUBEFLOW-NAME # {"$kpt-set":"name"}
+  name: containernodepool-gpu
+  namespace: KF-PROJECT # {"$kpt-set":"project"}
+spec:
+  location: LOCATION # {"$kpt-set":"location"}
+  initialNodeCount: 1
+  autoscaling:
+    minNodeCount: 0
+    maxNodeCount: 5
+  nodeConfig:
+    machineType: n1-standard-4
+    diskSizeGb: 100
+    diskType: pd-standard
+    preemptible: true
+    oauthScopes:
+    - "https://www.googleapis.com/auth/logging.write"
+    - "https://www.googleapis.com/auth/monitoring"
+    - "https://www.googleapis.com/auth/devstorage.read_only"
+    guestAccelerator:
+    - type: "nvidia-tesla-k80"
+      count: 1
+    metadata:
+      disable-legacy-endpoints: "true"
+  management:
+    autoRepair: true
+    autoUpgrade: true
+  clusterRef:
+    name: KUBEFLOW-NAME # {"$kpt-set":"name"}
+    namespace: KF-PROJECT # {"$kpt-set":"project"}
 ```
 
-Then create the [ContainerNodePool](https://cloud.google.com/config-connector/docs/reference/resource-docs/container/containernodepool) resource adopting GPU, for exmaple:
+Note that the `metadata:name` should be unique in your Kubeflow project. Because management cluster uses this as ID and your Google Cloud project as namespace to identify a nodepool.
 
-```
-  apiVersion: container.cnrm.cloud.google.com/v1beta1
-  kind: ContainerNodePool
-  metadata:
-    labels:
-      mesh_id: "proj-PROJECT_NUMBER" # {"$kpt-set":"asm-mesh-id"}
-    name: containernodepool-gpu
-  spec:
-    location: LOCATION # {"$kpt-set":"location"}
-    initialNodeCount: 1
-    autoscaling:
-      minNodeCount: 1
-      maxNodeCount: 5
-    nodeConfig:
-      machineType: n1-standard-4
-      diskSizeGb: 100
-      diskType: pd-standard
-      preemptible: false
-      oauthScopes:
-        - "https://www.googleapis.com/auth/logging.write"
-        - "https://www.googleapis.com/auth/monitoring"
-      guestAccelerator:
-        - type: "nvidia-tesla-k80"
-          count: 1
-      metadata:
-        disable-legacy-endpoints: "true"
-    management:
-      autoRepair: true
-      autoUpgrade: true
-    clusterRef:
-      name: "PROJECT/LOCATION/KUBEFLOW-NAME" # {"$kpt-set":"cluster-name"}
-```
+Apply the nodepool patch file above by running:
 
-You must also set 
-[`gpu-pool-initialNodeCount`](https://github.com/kubeflow/manifests/blob/4d2939d6c1a5fd862610382fde130cad33bfef75/gcp/deployment_manager_configs/cluster-kubeflow.yaml#L58).
+```bash
+kubectl --context=${MGMTCTXT} --namespace=${KF_PROJECT} apply -f <path-to-gpu-nodepool-file>
```

Comment:
Define all environment variables that your code samples use. Currently, I do not see the following environment variables defined: MGMTCTXT, KF_PROJECT, KF_NAME.

For example, KF_DIR is defined in the **Before you start** section.